cli/flags.go

```diff
@@ -84,19 +84,15 @@ error occurs in any statement, the command exits with a
 non-zero status code and further statements are not
 executed. Only the results of the first SQL statement in each
 positional argument are printed on the standard output.`),
-	"gossip": wrapText(`
-A comma-separated list of gossip addresses or resolvers for gossip
-bootstrap. Each item in the list has an optional type:
+	"join": wrapText(`
+A comma-separated list of addresses to use when a new node is joining
+an existing cluster. Each address in the list has an optional type:
```

Comment:
You may want to be more explicit about leaving it empty for the very first node. It's implied here, but emphasis wouldn't hurt.